### PR TITLE
EAGLE-271 Topology management in remote/local mode including start/stop operations

### PR DESCRIPTION
**Pull request by qingwen220**
**Related tasks** 
https://issues.apache.org/jira/browse/EAGLE-272
https://issues.apache.org/jira/browse/EAGLE-271
https://issues.apache.org/jira/browse/EAGLE-238

**Open issue**
EAGLE-263 Eagle tomcat server fail to shutdown for possible threads leak

### CODE REVIEW

**Comment by qingwen220** [comment]
The application manager tutorial is under eagle-docs/tutorial. Hope it can help.

File: eagle-assembly/src/main/conf/eagle-scheduler.conf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+### scheduler propertise
+appCommandLoaderInterval = 1
+appHealthCheckInterval = 5
```
Comment (RalphSu) [inline eagle-assembly/src/main/conf/eagle-scheduler.conf]: Better to have unit in name of config when relate to time.

**Comment by RalphSu** [comment]
Big PR. it's good to me unless the config item name.
+1

File: eagle-assembly/src/main/conf/eagle-scheduler.conf (hunk 1 above)
Comment (qingwen220) [inline eagle-assembly/src/main/conf/eagle-scheduler.conf]: Good suggestion. I will update it.

**Pull request closed without merging**